client/src/components/DataEntry/Ballot.tsx

```diff
@@ -73,14 +140,91 @@ const Ballot: React.FC<IProps> = ({
   )
   const ballot = ballots[ballotIx]
 
-  const [auditing, setAuditing] = useState(true)
   const [interpretations, setInterpretations] = useState<
     IBallotInterpretation[]
   >(contests.map(emptyInterpretation))
+  const { confirm, confirmProps } = useConfirm()
+
+  const interpretationsRef = useRef<IBallotInterpretation[]>(
+    contests.map(emptyInterpretation)
+  )
+  interpretationsRef.current = interpretations
```

Comment:
What's this ref for? Seems like an unnecessary duplicate of the interpretations state